### PR TITLE
Particularizes the term “BibTeX citations”

### PR DESCRIPTION
**Pull request by njbart**

### CODE REVIEW

File: docs/index.Rmd [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (yihui) [inline docs/index.Rmd]: Why was this line deleted?

File: docs/index.Rmd [hunk 2]
```diff
@@ -51,7 +50,7 @@ We introduce an R package, **blogdown**, in this short book, to teach you how to
...
+1. The website is generated from R Markdown documents (R is optional, i.e., you can use plain Markdown documents without R code chunks). This brings a huge amount of benefits, especially if your website is related to data analysis or (R) programming. Being able to use Markdown implies simplicity and more importantly, _portability_ (e.g., you are giving yourself the chance to convert your blog posts to PDF and publish to journals or even books in the future). R Markdown gives you the benefits of dynamic documents --- all your results, such as tables, graphics, and inline values, can be computed and rendered dynamically from R code, hence the results you present on your website are more likely to be reproducible. An additional yet important benefit of using R Markdown is that you will be able to write technical documents easily, due to the fact that **blogdown** inherits the HTML output format from **bookdown** [@xie2016]. For example, it is possible to write LaTeX math equations, citations from BibTeX, BibLaTeX, CSL, and other databases (using pandoc-citeproc), and even theorems and proofs if you want.
```
Comment (yihui) [inline docs/index.Rmd]: I'd just delete `BibTeX` in the original text to deemphasize it. Listing all kinds of bib databases sounds too detailed for the preface of the book...

Comment (njbart) [reply docs/index.Rmd]: That’s ok with me.
